cinema_service/settings.py

```diff
@@ -134,7 +140,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
It's recommended to use an absolute path for `MEDIA_ROOT`, you can use `os.path.join(BASE_DIR, 'media')` to achieve this.